interpreter/cling/tools/plugins/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+#------------------------------------------------------------------------------
+# CLING - the C++ LLVM-based InterpreterG :)
+#
+# This file is dual-licensed: you can choose to license it under the University
+# of Illinois Open Source License or the GNU Lesser General Public License. See
+# LICENSE.TXT for details.
+#------------------------------------------------------------------------------
+
+## If we drop a compatible cmake project in this folder we should automatically
+## pick it up and build it.
+#function(LISTSUBDIRS result curdir)
+#  file(GLOB children RELATIVE ${curdir} ${curdir}/*)
+#  set(dirlist "")
+#  foreach(child ${children})
+#    if(IS_DIRECTORY ${curdir}/${child})
+#      if(APPEND dirlist ${child})
+#    endif()
+#  endforeach()
+#  set(${result} ${dirlist} PARENT_SCOPE)
+#endfunction()
+#
+#LISTSUBDIRS(subdirs ${CMAKE_CURRENT_SOURCE_DIR}/plugins/)
+#foreach(subdir ${SUBDIRS})
+#  add_subdirectory(${subdir})
+#endforeach()
+
+ExternalProject_Add(
```

Comment:
Shouldn't this be in `clad/CMakeLists,txt`?

Reply:
Does this mean that an internet connection is always required to build ROOT when `clad` is enabled?

Reply:
Yes, the usual story as the other cases of `ExternalProject_Add`.

Reply:
Not really, as the other external projects can also be picked up from the system instead of being downloaded. I did not see such an option for clad. In any case, we already have many requests for making ROOT build without requiring an internet connection, and we even have [ROOT-9163](https://sft.its.cern.ch/jira/browse/ROOT-9163) to track progress towards it, and this goes in the opposite direction. That is, I'd rather not make it harder and harder to fix ROOT-9163 if I could.